scripts/bootstrapOrUpdate.ts

```diff
@@ -14,7 +14,7 @@ const csvFile = process.argv[2]
 export async function bootstrapOrUpdate(path: string) {
   const data = await convertCSVToJSON(path)
   try {
-    updateDatabase(data)
+    await updateDatabase(data)
```

Comment:
I found that `yarn update-database ./collections.csv` was actually just falling silently.

@mbilokonsky helped me see that we were blowing past this promise here. Not sure if this worked recently for others, maybe it's just been a while since anyone has tried this?

Reply:
Nice catch